### ISSUE TITLE
Q: Is it possible to add every new contact to a certain journey?

### ISSUE THREAD

**Issue by mattes3** [parcelvoy/platform#392]
When a new user signs up to one of my products, I want to send an `identify` call to Parcelvoy. Is it possible to add this new user to a certain journey, e.g. an "introduction" journey, automatically? If so, what do I need to do?

**Comment by pushchris** [comment]
@mattes3 apologies for the delay. There are a few ways to do what you are looking for:
1. Create a list with no rules and use that as the entrance to the journey (downside is it may enter all of your existing users right away)
2. Create an event alongside the identify call (obviously seems like you are trying to avoid this)
3. Use the `subscribed` event that is created when a user is subscribed to a given channel and look for the channel you care about 
4. Beta (will be out in next release): Switch to not identifying users but instead create an event and pass the user data into the event (there is a new field being added called `user` on that API call that will create the user if they do not exist). This code is merged but not yet in a release

Could also be worth on our end looking into adding events for account creation so that you could pull timestamp data from it, etc.

**Comment by mattes3** [comment]
Yes, I was trying to avoid the event because the introduction journey should be sent only once, regardless of how many ways my users sign up.

You mention a channel with a `subscribed` event. I didn't see a "channel" in my self-hosted Parcelvoy (maybe I overlooked it?). Can you share some detail about what a channel is and what it does? Or was it just a synonym for "list"?

**Comment by pushchris** [comment]
@mattes3 what do you mean by "how many ways a user signs up"? In theory the same user should share the same external ID value, so their profiles would be merged into one on Parcelvoy. When a user is added to a new subscription group (might be the better wording here) an event is triggered that they are now a part of that subscription. You could use the creation of that event to trigger the journey. Additionally, I merged some code in yesterday that will trigger a `user_created` event only when an account is first created which you could use for journeys as well

**Comment by mattes3** [comment]
The `user_created` event would already solve most of the problem, thanks so much! How do I get the new version, just a `docker pull` on the image with the `latest` tag?

"Many ways to sign up", yes that it is a special problem of mine (see also #398):

I am planning to create several signup forms for lead magnets, e.g. a free checklist for this, a blueprint for that, etc. So my prospects might sign up for many of these, and I want to start the introduction journey only once for each of them, even when they submit several signup forms.

**Comment by pushchris** [comment]
We are working on finishing up the checklist items for the next release which will be a fairly large one. Hoping to have it complete by tomorrow and will then add a new tag, until then you would unfortunately need to pull from source

**Comment by pushchris** [comment]
The event is now available in v1.4.0

**Comment by mattes3** [comment]
You mean `user_created` is fired? Thank you so much!
